gitops-wg/charter.md

```diff
@@ -0,0 +1,53 @@
+# GitOps Working Group Charter
+
+## Introduction
+
+This document outlines the charter for the GitOps Working Group, a [time-limited Working Group](https://github.com/cncf/toc/blob/main/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs) under the [CNCF App Delivery SIG](https://github.com/cncf/sig-app-delivery).
+
+## Mission Statement
+
+The focus of the GitOps WG is to clearly define a vendor-neutral, principle-led meaning of GitOps.
+This will establish a foundation for interoperability between tools, conformance, and certification. Lasting programs, documents, and code are planned to live within the [OpenGitOps](https://github.com/open-gitops) project.
+
+## Goals
+
+- Open governance
+- Project/vendor neutral
+- Promote awareness and understanding of GitOps terminology and approaches by creating education materials, organizing events and supporting practitioners throughout the community
+- Create documents and guidelines for following and implementing GitOps principles
+- Impartial analysis of the benefits and drawbacks of GitOps and its various approaches
+- Maintain a vendor-neutral landscape of GitOps projects/products and solutions
+
+## Non-Goals
+
+- Recommend/endorse specific projects, vendors or approaches to GitOps
+- Exclude/deprecate specific projects, vendors or approaches to GitOps
+- Promote GitOps over other paradigms or methodologies (i.e. no advocacy or branding)
+
+## Work Products (WIP)
+
+- GitOps definition and terminology document: working draft
+- GitOps methodologies and approaches document
+- GitOps landscape document
+- GitOps community events (e.g., GitOpsCon)
+- Working examples for getting started
+- OpenGitOps project website
+
+## Cloud Native Alignment
+
+The GitOps WG works to enable widespread and successful adoption of the GitOps pattern in cloud-native environments by:
+
+- Providing valuable deliveries to the GitOps community, and objective information to CNCF TOC, End Users and Projects of the CNCF regarding areas considered in scope (see [goals](#goals) and [non-goals](#non-goals))
+- Collaborating effectively with [other related groups](#other-related-groups)
+- Helping to maintain the continued health of GitOps related projects in CNCF
+- Identifying and filling gaps in the landscape of GitOps Projects
+
+## SIG App Delivery Alignment
+
+As an “operating model” for cloud native applications, GitOps fully aligns with "Application delivery workflow and strategy" and "Configuration source driven workflow" scopes defined in CNCF SIG App Delivery charter.
+
+## Other Related Groups
+
+- CNCF SIG-Security: Security in GitOps workflow is one of the most important areas in consideration
+- CNCF SIG-Runtime: The execution engine of GitOps workflow is mostly a controller inside Kubernetes which drops in scope of SIG Runtime
+- Continuous Delivery Foundation (CDF) cdCon GitOps Summit
```

Comment:
Maybe also add the cooperative delivery WG?

Reply:
Yes 👍

Reply:
Added